common/namespace/handler.go

```diff
@@ -455,6 +443,12 @@ func (d *HandlerImpl) UpdateNamespace(
 			// only do merging
 			info.Data = d.mergeNamespaceData(info.Data, updatedInfo.Data)
 		}
+		if updatedInfo.State != enumspb.NAMESPACE_STATE_UNSPECIFIED && info.State != updatedInfo.State {
```

Comment:
when will state be `enumspb.NAMESPACE_STATE_UNSPECIFIED`?

Reply:
If it is not changed (not specified in tctl command).